README.md

```diff
@@ -3,47 +3,60 @@
 
-You can configure the following properties with `-D`, e.g., `-DauthUsers=50`. Defaults are in the `pom.xml`.
+1. The tests will still run to completion regardless; if there are failures, there will a message so indicating at the end.
+2. At lower loads, the tests will typically all pass (TRS ones may time out on a local instance); failures start happening when the load 
+becomes too high.
 
-* authUsers -- the number of authenticated users to simulate; defaults to 50
-* anonUsers -- the number of anonymous users to simulate; defaults to 50; only applies if scenario, below, is `Everything`
-* atOnce -- true if all users should hit at once, or if they should ramp up over time; defaults to false
-* rampMinutes -- if `atOnce` is not true, the number of minutes the specified number of users will be phased in; defaults to 5
-* baseUrl -- the Dockstore webservice endpoint to run the tests against; defaults to `http://localhost:8080`
-* scenario -- the name of the scenario to run; see DockstoreWebUser.scala for all available; defaults to `Everything`, which runs (almost) everything
-* maxResponseTimeMs -- if any API call takes longer than this, simulation will fail; default is 10,000, which is probably too high
-* successThreshold -- the percentage of calls that should pass; if less, the simulation fails; default is 95
-* percentToPublish -- the percentage of workflows created in the `HostedWorkflowCrud` scenario that will be published; the default is 25
+#### Simulation Scenarios
+
+1. Terra Requests: Simulates a workflow running in Terra, that fetches the primary descriptor. The rate is controlled by the `terraRequestsRps` property, defaults to 8 per second. Fetches a WDL descriptor files that are specified in `data/workflows.csv`.
+2. Terra Version: Fetches a descriptor's versions, currently controlled by `webSiteUsers` (a Terra workflow run fetches a descriptor's versions a couple of times)
+3. Web site users: Simulates a non-authorized user going through the web site, home page, search, go to a workflow, download its zip. The number of users, controlled by the `webSiteUsers` property, is spread out over the duration of the run. It defaults to 6.
+4. Trs : Simulates requests being made to fetch all TRS tools and then fetching a tool, using both TRS v1 and v2. And makes the calls Galaxy search uses. Controlled by `trsRequestsPerHour`. 
+5. GitHub Notifications - Invokes the GitHub release API. Invokes them against the repositories in `data/githubRefresh.csv`. Note this scenario will only be run if `installationId` and `curatorToken` are both
+set, otherwise it is skipped.
 
-Regarding the last two items, the tests will still run to completion; if there is failure, there will a message so indicating at the end.
+#### Curator token and Installation Id
 
-The default values are defined in the `<properties>` section of the pom.xml.
+The tests can invoke the `/workflows/github/release` endpoint, the one invoked indirectly invoked by GitHub Apps from
```

Comment:
I think there's an extra invoked in there that's not needed

Reply:
It was intended, but I'll make it clearer.

21425cf556789c75b4c63d231642d91fd39cedf4